src/components/stepWizard/stepWizardContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useCallback, useState, ReactNode } from "react";
```

Comment:
I would create a new directory called ```store``` and put this file into it. All contexts will be stored in the ```store``` directory in the future